app/konnect/dev-portal/access-and-approval/add-teams.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Add Developer Teams from Identity Providers
+content_type: how-to
+---
+
+In the {{site.konnect_short_name}} Dev Portal, you can import existing developer teams from a third-party identity provider (IdP) and map their permissions to elements in {{site.konnect_short_name}}. This allows you to import developers automatically instead of manually copying over every single team of developers from your IdP to {{site.konnect_short_name}}.
+
+This guide explains how to map the permissions, including scopes and claims, from your group of developers in your IdP to your organization's team in {{site.konnect_short_name}}. Although this guide uses Okta, Azure Active Directory (AD), and Auth0 as examples, you can use any IdP that conforms to OIDC standards. 
+
+## Prerequisites
+
+* A test developer account in your IdP
+* An IdP configured in Konnect (OIDC or DCR) for dynamic client registration:
+    * [Okta](/konnect/dev-portal/applications/dynamic-client-registration/okta/)
+    * [Auth0](/konnect/dev-portal/applications/dynamic-client-registration/auth0/)
+    * [Azure AD](/konnect/dev-portal/applications/dynamic-client-registration/azure/)
+* [Enable app registration](/konnect/dev-portal/applications/enable-app-reg/) for Dev Portal
+
+## Create developer teams in your IdP
```

Comment:
@c3pko I adjusted a few of the doc links because I think I got them wrong. Do you have a dev on your team that could look this new page for accuracy? I'm just really unsure if I missed any set up steps in the IdP. 

I know we're running low on time if the plan is to release this next week, so I can always comment out this section and add "Groups created in your IdP" as a prerequisite to this doc. That way we can do a thorough tech review post-release and then add this section in then.

Reply:
I'm not sure all three of these Okta mapping steps are required. Let's just keep the team mapping in IDP step as a one liner of "configure your teams in your IdP" with the right links out to Okta/Azure/Auth0. We can go back in later and add team specific steps for each IdP.